### PR TITLE
t/2020 fall schedule import mods

### PR DESCRIPTION
**Pull request by nataliemona**

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -608,35 +608,54 @@ def lower_case_first_line(iter):
...
+    for col in required_columns:
+        if col not in rows[0]:
+            raise Exception(f"Missing required field \"{col}\"")
+
     schedule_data = {}
     for row in rows:
 
+        meeting_id_with_pwd = row["meeting id with password"]
+        if not meeting_id_with_pwd:
+            print(f"{row['course code']}: \tMissing zoom link")
```
Comment (nataliemona) [inline tasks.py:627]: helpful to have separate errors for missing link vs invalid link

File: tasks.py [hunk 2]
```diff
@@ -608,35 +608,54 @@ def lower_case_first_line(iter):
...
+            continue
+
         try:
-            zoom_link = urlparse(row["meeting id with password"])
+            zoom_link = urlparse(row["meeting id with password"].strip())
```
Comment (nataliemona) [inline tasks.py:631]: some links were marked invalid for having spaces around them

File: tasks.py [hunk 3]
```diff
@@ -608,35 +608,54 @@ def lower_case_first_line(iter):
...
+            print(f"{row['course code']}: \tInvalid zoom link")
 
         zoom_series_id = zoom_link.path.split("/")[-1]
         schedule_data.setdefault(zoom_series_id, {})
         schedule_data[zoom_series_id]["zoom_series_id"] = zoom_series_id
 
         opencast_series_id = urlparse(row["oc series"]) \
             .fragment.replace("/", "")
+        if not opencast_series_id:
+            print(f"{row['course code']}: \tMissing oc series")
```
Comment (nataliemona) [inline tasks.py:643]: oc series should be required

File: tasks.py [hunk 4]
```diff
@@ -608,35 +608,54 @@ def lower_case_first_line(iter):
...
 
         subject = "{} - {}".format(row["course code"], row["type"])
         schedule_data[zoom_series_id]["opencast_subject"] = subject
-        
+      
         schedule_data[zoom_series_id].setdefault("Days", set())
-        for day in row["day"].strip():
+        split_by = " "
+        if "," in row["day"]:
+            split_by = ","
```
Comment (nataliemona) [inline tasks.py:652]: handle several possible cases in the "Day" field...
"MW"
"M,W"
"M W"
"M, W"

File: tasks.py [hunk 5]
```diff
@@ -608,35 +608,55 @@ def lower_case_first_line(iter):
...
+        days = [day.strip() for day in row["day"].split(split_by)]
+        for day in days:
             if day not in valid_days:
-                raise Exit("Got bad day value: {}".format(day))
+                print(
+                    f"{row['course code']}: \tbad day value \"{day}\""
```
Comment (nataliemona) [inline tasks.py:657]: Match format with other print statements.

**Review by lbjay — APPROVED** [review]

**Pull request merged**